spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/DefaultBinderFactory.java

```diff
@@ -57,7 +57,7 @@
 
 	private volatile ConfigurableApplicationContext context;
 
-	private Map<String, String> defaultBindersPerBoundType = new HashMap<>();
+	private Map<String, String> defaultBinderForBindingTargetType = new HashMap<>();
```

Comment:
actually this being singular doesn't sound like a collection... why not just "defaultBinders"? (with a comment that mentions the key of the map is the type)